samples/HelloWorld/HelloWorld/HelloWorld.DotNet.csproj

```diff
@@ -10,4 +10,19 @@
   <ItemGroup>
     <Compile Remove="Properties\AssemblyInfo.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference
+        Include="Microsoft.Intune.MAM.Remapper.Tasks"
+        Version="0.1.4635.1"
+        IncludeAssets="none"
+        GeneratePathProperty="True"
+        ReferenceOutputAssembly="False"
+    />
+  </ItemGroup>
+  <Target Name="_AddMamFiles"
+      BeforeTargets="_AddAndroidCustomMetaData">
+    <ItemGroup>
+      <_AndroidMamMappingFile Include="$(PkgMicrosoft_Intune_MAM_Remapper_Tasks)/content/MonoAndroid10/remapping-config.json" />
+    </ItemGroup>
+  </Target>
```

Comment:
Should we really put this in the Hello World project? Should it maybe be an MSBuild test in Xamarin.Android.Build.Tests?

Reply:
Beginnings of a "lesser" test just pushed, via introduction of a new `@(_AndroidRemapMembers)` build action.  This uses the same XML format that the InTune JSON is converted to, so we can test things by writing our XML format directly, instead of going through a JSON intermediary.

TODO: add an actual on-device unit test which does this.